es-app/src/SystemScreenSaver.cpp

```diff
@@ -4,6 +4,7 @@
 #endif
 #include "components/VideoVlcComponent.h"
 #include "platform.h"
+#include "PowerSaver.h"
```

Comment:
Again, these changes are not part of GridView feature improvements, but likely the result of manual merging of commits to RetroPie/master after your branching point.
Please rebase the branch, to clean these up.

(Git is a pain, I know).